docs/example-react-hooks.md

```diff
@@ -3,9 +3,7 @@ id: example-react-hooks
 title: React Hooks
 ---
 
-`react-testing-library` provides the
-[`testHook`](/docs/react-testing-library/api#testhook) utility to test custom
-hooks.
+[`react-hooks-testing-library`](https://github.com/mpeyper/react-hooks-testing-library) is built on top of [`react-testing-library`](/react) to create a simple test harness for [React hooks](https://reactjs.org/docs/hooks-intro.html) that handles running them within the body of a function component, as well as providing various useful utility functions for updating the inputs and retrieving the outputs of your [custom hook](https://reactjs.org/docs/hooks-custom.html).
```

Comment:
we should replace this description with react-hooks-testing-library ["The solution"](https://github.com/testing-library/react-hooks-testing-library#the-solution) section with an updated link for [the react-hooks-testing-library](https://github.com/testing-library/react-hooks-testing-library) and [react-testing-library](https://github.com/testing-library/react-testing-library)